chronos/transports/stream.nim

```diff
@@ -1734,19 +1753,22 @@ else:
                 trackStream(ntransp)
                 retFuture.complete(ntransp)
               else:
-                retFuture.fail(getTransportOsError(osLastError()))
+                let errorMsg = osErrorMsg(osLastError())
+                retFuture.fail(getConnectionAbortedError(errorMsg))
             else:
-              let err = osLastError()
-              if int(err) == EINTR:
+              let err = int(osLastError())
+              if err == EINTR:
                 continue
-              elif int(err) == EAGAIN:
+              elif err == EAGAIN:
```

Comment:
EWOULDBLOCK as well.

Reply:
EWOULDBLOCK should be never happened, even if it mentioned in manual `EWOULDBLOCK` means that async framework is not working, unable to properly deliver events that socket is ready for non-blocking operation.